pkg/filter/evaluators.go

```diff
@@ -0,0 +1,149 @@
+package filter
+
+import (
+	"fmt"
+	"reflect"
+	"regexp"
+)
+
+const (
```

Comment:
Why don't we use standard wording for this? Equal and NotEqual?

Reply:
Thanks for the suggestion :+1: